alf/utils/normalizers.py

```diff
@@ -122,8 +122,7 @@ def _summary(name, val):
                 with alf.summary.scope(self._name):
                     if val.ndim == 0:
                         alf.summary.scalar(name + "." + suffix, val)
-                    elif (val.shape[0] < self.MAX_DIMS_TO_OUTPUT
-                          and alf.summary.should_summarize_output()):
```

Comment:
It's not easy to relate ``should_summarize_output()`` with this function here without looking at the source code, so I just removed this check.

Reply:
Just be aware that it could potentially dump a lot of tb plots, multiple for each dimension of observation.  It will slow down training and clog hard disk if ran many times.

Reply:
> Just be aware that it could potentially dump a lot of tb plots, multiple for each dimension of observation. It will slow down training and clog hard disk if ran many times.

It's kind of confusing when setting ``debug_summaries=True`` alone won't actually summarize, without turning on ``should_summarize_output``. Should we just decrease the value self.MAX_DIMS_TO_OUTPUT?

Reply:
If we make MAX_DIMS_TO_OUTPUT configurable, then we can decrease the default value.

Reply:
> If we make MAX_DIMS_TO_OUTPUT configurable, then we can decrease the default value.

I've changed this parameter to an argument.